bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(
+            filename, file, config_file,
+            use_spaces: bool = False):
+
+        if use_spaces:
+            raise ValueError(
+                'Set "use_spaces=False" for HappinessLintBear '
+                'to avoid conflicts with other bears'
```

Comment:
still not the reason, you don't get conflicts with other bears, the underlying linter just always wants to use tabbed indentation, that's why we want to throw an error if a user has this setting set to something else.